tests/test_merge_IndexFlatCodes.py

```diff
@@ -0,0 +1,39 @@
+# Test merge_from method for all IndexFlatCodes Types
```

Comment:
Don't make such small test files. 
Please call it test_index_merge.py and move all the merge testing logic here (including the IVF).